azure-pipelines-e2e-tests-template.yml

```diff
@@ -20,17 +20,24 @@ stages:
       - job: Run_${{ replace(option.pool, ' ', '_') }}
         ${{ if eq(lower(option.pool), 'windows') }}:
           pool:
-            vmImage: 'windows-2022'
+            name: Cosmos2MT-AzureDevOps-AgentPool
```

Comment:
Can it have a different name? Cosmos2MT is kind of internal name.

Reply:
It's the only one available Pool for our team in ManagedOSS project, other pipelines also share the pool, the name can't be changed, or we request to create new agent pool. I think it's OK for the name as the pipeline is only for internal purpose, if customer need to run pipeline based on the yaml file, they need to change to their own pool anyway.

Reply:
OK